include/swift/AST/DiagnosticsSema.def

```diff
@@ -7523,13 +7523,6 @@ ERROR(result_builder_buildpartialblock_accumulated_not_accessible,none,
         "expression shuffles the elements of this tuple; "
         "this behavior is deprecated", ())
 
-//------------------------------------------------------------------------------
-// MARK: Implicit conversion diagnostics
-//------------------------------------------------------------------------------
-ERROR(cannot_implicitly_convert_in_optional_context,none,
```

Comment:
For posterity: this is going to come back because it's still a useful warning to have.